contents/plenum/2020-10-20/_index/main.md

```diff
@@ -44,6 +53,7 @@
 
 ### Memberships
 - (Bob) Update on new members (if any).
+- (Valentine) Request for latest membership breakdown by tiers.
```

Comment:
@hyperair For your action.

Reply:
Tier | 16 | 32 | 64 | 128 | 256 | 512
-- | -- | -- | -- | -- | -- | --
Count | 4 | 0 | 13 | 12 | 1 | 3

2 of the `512`'s will probably drop by December.